src/daft-core/src/python/series.rs

```diff
@@ -153,6 +153,22 @@ impl PySeries {
         Ok(self.series.sign()?.into())
     }
 
+    pub fn signum(&self) -> PyResult<Self> {
+        Ok(self.series.signum()?.into())
+    }
+
+    pub fn negate(&self) -> PyResult<Self> {
+        Ok(self.series.negate()?.into())
+    }
+
+    pub fn negative(&self) -> PyResult<Self> {
```

Comment:
same comment here. lets not add multiple implementations (aliases) in the core, only on the frontends (dataframe, sql, spark)